src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const checkSelectedId = tab => {
+    if (tab.id !== currentTab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={currentTab.id === tab.id ? 'is-active' : ''}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally